addons/elasticsearch/6.8.x/elasticsearch-2.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: elasticsearch
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: elasticsearch
+    # TODO: we're temporarily supporting dependency on an existing default storage class
+    # on the cluster, this hack will trigger re-queue on Addons until one exists.
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.2-2"
+    appversion.kubeaddons.mesosphere.io/elasticsearch: "6.8.2"
+    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/helm/charts/6bfbc8018cd4440637b07c7559d5812e4d9db34d/stable/elasticsearch/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: true
+  chartReference:
+    chart: stable/elasticsearch
+    version: 1.32.0
+    values: |
+      ---
+      client:
+        heapSize: 1024m
+        resources:
+          limits:
+            cpu: 500m
+            memory: 2048Mi
+          requests:
+            cpu: 100m
+            memory: 1536Mi
+      master:
+        updateStrategy:
+          type: RollingUpdate
+        heapSize: 1024m
+        resources:
+          # need more cpu upon initialization, therefore burstable class
+          limits:
+            cpu: 1000m
+            memory: 2048Mi
+          requests:
+            cpu: 500m
+            memory: 1536Mi
+      data:
+        updateStrategy:
+          type: RollingUpdate
+        hooks:
+          drain:
+            enabled: false
```

Comment:
Setting `data.hooks.drain.enabled: false` removes these pre-stop and post-start hooks: https://github.com/helm/charts/blob/2fe0e135074c840b08f7517aa642b857e6a519a4/stable/elasticsearch/templates/configmap.yaml#L104-L140

The pre-stop hook looks important, it migrates data to prevent it from being lost after the node shuts down. And the post-start hook appears to add the new node to elasticsearch client config (I'm not sure how we're getting by without it). Should we add those scripts back in the hooks we're setting?

Reply:
I did not set this value. And honestly I do not know who made that decision. This does not turn off the hooks, it just keeps a default set of hooks from running. I suggest that if we wish to enable that we can write it in full here while setting that to false, otherwise we will have to maintain our own chart.

Reply:
My mistake, I got the impression from the following comment that this PR was setting this value. If we aren't changing anything here them I'm not concerned.

Reply:
Yeah, those comments arent very clear and it took me a few readings plus trials to fully grasp what the intents were. Anyway thanks for looking in and challenging this, always good to double check understanding :)